metastore/schema_fetch.go

```diff
@@ -201,7 +201,7 @@ func (j *SchemaFetchJob) FetchEnum() {
 			continue
 		}
 		for _, column := range table.Columns {
-			if !column.IsEnumColumn() {
+			if !column.IsEnumBasedColumn() {
```

Comment:
`!column.IsEnumBasedColumn() and !column.ISEnumArrayColumn()` ?

Reply:
! IsEnumBasedColumn already covered both cases